src/legacy/store/GraphQLStoreChangeEmitter.js

```diff
@@ -22,11 +21,10 @@ export type ChangeSubscription = {
   remove: () => void;
 };
 
-var batchUpdate = callback => callback();
-var subscribers = [];
-
-var executingIDs = {};
-var scheduledIDs = null;
+type Subscriber = {
+  callback: Function,
```

Comment:
nit: it's always better to be as detailed as possible with types, in this case the callback is `() => void`.